src/core/Modal/Modal.tsx

```diff
@@ -100,6 +105,7 @@ export const Modal = (props: ModalProps) => {
     className,
     style,
     children,
+    styleType: type,
```

Comment:
Why casting to another name?

Reply:
I think this happened when I changed the name. Fixed.